.github/workflows/bot-auto-merge.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: bot-auto-merge
+
+on:
+  workflow_run:
+    types: [completed]
+    workflows: [tox-pytest]
+
+jobs:
+  bot-auto-merge:
+    name: Auto-merge passing bot PRs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Auto-merge passing dependabot PRs
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        uses: ridedott/merge-me-action@v2
+        with:
+          # For clarity only. dependabot is default.
+          GITHUB_LOGIN: dependabot
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ENABLED_FOR_MANUAL_CHANGES: "true"
```

Comment:
So ` ridedott/merge-me-action@v2` only merges PRs created by the bot in `GITHUB_LOGIN`, correct?

Reply:
Yes that's right. This step will only affect `dependabot` PRs. And the other one will only affec `pre-commit-ci` PRs.